### ISSUE TITLE
Remove the titlebar, for more space…

### ISSUE THREAD

**Issue by bwinton** [bwinton/TabCenter#717]
It should look something like this on macOS:
<img width="1156" alt="screenshot 2016-10-20 11 07 24" src="https://cloud.githubusercontent.com/assets/159571/19567408/83fece88-96bb-11e6-80b7-9788c42b465d.png">
(Added `#verticaltabs-box { padding-top: 32px; }` in the Inspector.)

and something like this on Windows:
<img width="1217" alt="screenshot 2016-10-20 15 12 08" src="https://cloud.githubusercontent.com/assets/159571/19574287/9b619dbe-96d7-11e6-8da6-c617b81916d1.png">
(Added `#nav-bar { margin-right: 138px; }` in the Inspector.)

But if @phlsa or @shorlander disagree, go with their suggestions instead…  :wink:

**Comment by ChrisW-B** [comment]
I've been maintaining a version similar to this in my spare time, if that helps. While the Windows version is exactly the same, I elected to shift the tab center down instead when not in fullscreen for OS X ![screen shot 2016-10-21 at 10 14 59 am](https://cloud.githubusercontent.com/assets/2490974/19601435/564a651c-9777-11e6-9bec-2192bd8c10ef.png)

its here if you want to look at it [ChrisW-B/TabCenter](https://github.com/ChrisW-B/TabCenter)

**Comment by bwinton** [comment]
Yeah, @phlsa has been adamantly against putting the urlbar above the tabs (for similar reasons to us having the tabs above the urlbar in the regular layout), so we're probably not going to go in that direction…  But thanks for the pointer!  🙂

**Comment by ChrisW-B** [comment]
I made a quick version of it by adding 
```css
#main-window:not([inFullscreen]):not([inDOMFullscreen]) #verticaltabs-box {
  padding-top:25px!important;
}
```
to OSX.css
![screen shot 2016-10-21 at 8 38 25 pm](https://cloud.githubusercontent.com/assets/2490974/19615560/d85b795e-97ce-11e6-88ce-723c97448a30.png)

But I think the main problem you'd have is what to do with the collapsed version
![screen shot 2016-10-21 at 8 38 34 pm](https://cloud.githubusercontent.com/assets/2490974/19615559/d671a8c0-97ce-11e6-8295-2964f501460e.png)
If you were to go through removing the titlebar, I'd guess that you'd have to either extend the navigator toolbox backwards, make the collapsed section larger, or have some short of tetris/upside down L shaped section

**Comment by JonasCz** [comment]
Using it with Hide Caption Titlebar Plus on Linux works quite well, too:

![screenshot at 2016-10-22 08 24 20](https://cloud.githubusercontent.com/assets/10514959/19617452/f8509ef8-9830-11e6-91a6-ec8d1be6e215.png)

**Comment by aleth** [comment]
It would be nice to see one of these solutions adopted, as otherwise the vertical space saving gained by having the tabs on the side is mostly wasted.
